__tests__/app.test.js

```diff
@@ -74,10 +55,70 @@ describe("GET /api" , () => {
             })
         })
     })
+})
+
+/*
+
+
+
+Should:
+
+be available on /api/articles/:article_id.
+get an article by its id.
+Responds with:
 
-    it("should respond with 404 upon invalid request", () => {
+an article object, which should have the following properties:
+author
+title
+article_id
+body
+topic
+created_at
+votes
+article_img_url
+Consider what errors could occur with this endpoint, and make sure to test for them.
+
+Remember to add a description of this endpoint to your /api endpoint.
+
+*/
+
+
+describe("GET /api/articles/:article_id", () => {
+    it("should respond with 200 and the correct article object", () => {
+        const articleId = 3
         return request(app)
-        .get("/invalid")
-        .expect(404)
+        .get(`/api/articles/${articleId}`)
+        .expect(200)
+        .then(({ body }) => {
+            expect(body.article[0]).toEqual(expect.objectContaining({
+                author: expect.any(String),
+                title: expect.any(String),
+                article_id: expect.any(Number),
+                body: expect.any(String),
+                topic: expect.any(String),
+                created_at: expect.any(String),
+                votes: expect.any(Number),
+                article_img_url: expect.any(String),
+            }))
+            expect(body.article[0].article_id).toBe(articleId)
+        })
```

Comment:
This test suggests that the ```body``` is an array. As you are only sending a single article, it should be an object. You can also explicitly assert the article id in the object instead of ```expect.any``` So it can be - ```article_id: 3```